src/main/java/seedu/progresschecker/logic/parser/AnswerCommandParser.java

```diff
@@ -0,0 +1,42 @@
+package seedu.progresschecker.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.progresschecker.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.progresschecker.commons.exceptions.IllegalValueException;
+import seedu.progresschecker.logic.commands.AnswerCommand;
+import seedu.progresschecker.logic.parser.exceptions.ParseException;
+import seedu.progresschecker.model.exercise.QuestionIndex;
+import seedu.progresschecker.model.exercise.StudentAnswer;
+
```

Comment:
do you wanna indicate the "author" here?

Reply:
Oh ok thanks!